pom.xml

```diff
@@ -1871,6 +1872,25 @@
         <artifactId>paranamer</artifactId>
         <version>${paranamer.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-vector</artifactId>
+        <version>${arrow.version}</version>
+        <exclusions>
```

Comment:
Should we consider explore excluding netty here, since we exclude it in most of the other related projects (like parquet), since it seems to have added some unnecessary jars to the deps list.